Shared/Samples/Identify layer features/IdentifyLayerFeaturesView.swift

```diff
@@ -0,0 +1,182 @@
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct IdentifyLayerFeaturesView: View {
+    /// The view model for the sample.
+    @StateObject private var model = Model()
+    
+    /// The tapped screen point.
+    @State var tapScreenPoint: CGPoint!
+    
+    var body: some View {
+        ZStack {
+            MapViewReader { proxy in
+                MapView(map: model.map)
+                    .onSingleTapGesture { screenPoint, _ in
+                        tapScreenPoint = screenPoint
+                    }
+                    .task {
+                        do {
+                            // Add map image layer.
+                            let mapImageLayer = ArcGISMapImageLayer(url: .worldCities)
+                            try await mapImageLayer.load()
+                            
+                            // Hide continent and world layers.
+                            mapImageLayer.subLayerContents[1].isVisible = false
+                            mapImageLayer.subLayerContents[2].isVisible = false
+                            model.map.addOperationalLayer(mapImageLayer)
+                            
+                            // Add feature layer.
+                            let featureTable = ServiceFeatureTable(url: .damageAssessment)
+                            try await featureTable.load()
+                            let featureLayer = FeatureLayer(featureTable: featureTable)
+                            model.map.addOperationalLayer(featureLayer)
+                        } catch {
+                            model.error = error
+                        }
+                    }
+                    .task(id: tapScreenPoint) {
+                        // Identify layers using the screen point.
+                        if let screenPoint = tapScreenPoint,
+                           let results = try? await proxy.identifyLayers(
+                            screenPoint: screenPoint,
+                            tolerance: 12,
+                            returnPopupsOnly: false,
+                            maximumResultsPerLayer: 10) {
+                            model.handleIdentifyResults(results)
+                            model.isShowingLayerAlert = true
+                        }
+                    }
+                    .alert(isPresented: $model.isShowingLayerAlert) {
+                        Alert(
+                            title: Text(model.layerAlertTitle),
+                            message: Text(model.layerAlertMessage)
+                        )
```

Comment:
Here I'd like to stray from the design to not use an alert, as it is possible that Apple/SwiftUI would move away from this iOS specific UI to sth else in the future. https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/issues/348

Instead, can you use a overlay status bar at the top, like other samples? Also adjust the README a bit. Thanks.

Reply:
Also, `Alert` is deprecated. While the [documentation](https://developer.apple.com/documentation/swiftui/view/alert(ispresented:content:)) currently says that it was deprecated in iOS 17, the suggested replacement modifier was introduced in iOS 15. That's actually when `Alert` was deprecated. Since the replacement API is available on every OS that we support, we shouldn't be using the deprecated API at all.